py-polars/tests/unit/operations/test_interpolate_by.py

```diff
@@ -86,10 +86,11 @@ def test_interpolate_by(
         .sort("times")
         .drop("times")
     )
-    assert_frame_equal(result, expected)
+    # assert_frame_equal(result, expected)
```

Comment:
why is this commented out?

Reply:
Probably by mistake. Good catch. I'll do the other clean ups (though what do you mean by missing full stop?)

Thanks. And let me know what you want to do with the name of the parameter.